contracts/veDistributionSnapshot.sol

```diff
@@ -56,21 +56,28 @@ contract veDistributionSnapshot is Ownable, ReentrancyGuard {
         return users;
     }
 
-    function withdrawReward(IERC20[] calldata tokens_) public {
+    function availableReward(address user_, IERC20 token_) public view returns (uint) {
+        uint userBalance = balanceOf[user_];
+        if (userBalance == 0) {
```

Comment:
Perhaps, zero should be returned also during the snapshot population, since rewards may jump misleadingly during that process.